hack/run-e2e-kind.sh

```diff
@@ -30,17 +30,28 @@ export CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
 
 export KIND_OPT=${KIND_OPT:="--config ${VK_ROOT}/hack/e2e-kind-config.yaml"}
 
-function get-k8s-server-version {
-    echo $(kubectl version --short=true | grep Server | sed "s/.*: v//" | tr "." " ")
+function check-jq {
+    echo "Checking jq"
+    which jq >/dev/null 2>&1
+    if [[ $? -ne 0 ]]; then
+        echo "Installing jq ..."
+        wget https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-1.7.1.tar.gz
+        tar -xf jq-1.7.1.tar.gz && cd jq-1.7.1
+        ./configure
+        make && sudo make install
+        rm -rf jq-1.7.1 jq-1.7.1.tar.gz
+    else
+        echo -n "Found jq, version: " && jq --version
+    fi
 }
 
 function install-volcano {
   install-helm
+  check-jq
 
   # judge crd version
-  serverVersion=($(get-k8s-server-version))
-  major=${serverVersion[0]}
-  minor=${serverVersion[1]}
+  major=$(kubectl version --output=json |jq '.serverVersion.major' | tr "\"" " ")
```

Comment:
kubectl version --output yaml | awk '/serverVersion/,0' | grep -E 'major:' | awk '{print $2}' | tr "\"" " "

Reply:
very good. I have remade the PR without jq.